app/tests/test_record.py

```diff
@@ -251,6 +251,66 @@ def test_record_record_arrangement(
             {"class": "record-container"},
         )
 
+    @mock_aws
+    def test_record_record_alert_banner_is_visible_when_unsupported_file_extension(
+        self, app, client: FlaskClient, mock_standard_user, record_files
+    ):
+        """
+        Given a File in the database
+        When a standard user with request to view the record details page
+        Then the response status code should be 200
+        If the extension of the File requested is not compatible with IIIF
+        Then the alert banner responsible with alerting the user should be visible
+        """
+
+        # index 6 is .foobar
+        file = record_files[6]["file_object"]
```

Comment:
for example here we just need one file, so dont necessarily need to reference record_files